.github/workflows/pythonapp.yml

```diff
@@ -38,4 +38,4 @@ jobs:
         if: ${{ github.repository == 'FEniCS/fiat' && github.head_ref == '' && matrix.os == 'ubuntu-latest' && matrix.python-version == '3.8' }}
         env:
           COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
-        run: coveralls
+        run: coveralls
```

Comment:
This file somehow does not want to disappear from the diff...

Reply:
Trying again with a newline but not a blank one.